src/storage/exec/StoragePlan.h

```diff
@@ -79,6 +79,13 @@ class StoragePlan {
         CHECK_LT(idx, nodes_.size());
         return nodes_[idx].get();
     }
+    std::vector<RelNode<T>*> getNodes() {
```

Comment:
Could just return nodes_'s const reference is ok.

`getNode` method is only used in test case.